src/seedu/addressbook/data/person/Name.java

```diff
@@ -39,10 +39,10 @@ public static boolean isValidName(String test) {
     /**
      * Retrieves a listing of every word in the name, in order.
      */
-    public List<String> getWordsInName() {
-        return Arrays.asList(fullName.split("\\s+"));
-    }
-
+    //implement interface printable
+    public static Printable getPrintableString( Printable fullName) {
```

Comment:
This time you are returning a value that you have passed in as a method argument. Which is incorrect as well. 
What is required is for you to return a string value of the value assigned to this object type upon instantiation. 

I highly recommend you to show this PR to one of your teammates and discuss it along with my comments. 
If you are still not clear about what is happening please talk to me further.